info.json

```diff
@@ -28,26 +28,67 @@
             }
         ],
         "modules": {
-            "threat_intel_feeds": [],
-            "workspaces":[]
+            "attachments": {
```

Comment:
please remove attachments, indicators and tasks. any m2m with workspace module will be captured from the workspace side. need not package these related modules

Reply:
removed unwanted module schema's